src/Sylius/Bundle/AdminBundle/Tests/MessageHandler/Admin/SendResetPasswordEmailHandlerTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ *  This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Tests\MessageHandler\Admin;
+
+use Sylius\Bundle\CoreBundle\Message\Admin\Account\SendResetPasswordEmail;
+use Sylius\Bundle\CoreBundle\MessageHandler\Admin\Account\SendResetPasswordEmailHandler;
+use Sylius\Component\Core\Model\AdminUser;
+use Sylius\Component\Core\Test\Services\EmailCheckerInterface;
+use Sylius\Component\Mailer\Sender\SenderInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class SendResetPasswordEmailHandlerTest extends KernelTestCase
```

Comment:
I'm wondering do we need this test 🤔 It's well covered with Behat scenarios, as far as I see. It also has spec already. So we can safely skip it IMO 🖖 (even though generally more tests -> better 🔥)

Reply:
Behat is testing the application as a whole (with ApiBundle, ShopBundle and AdminBundle). I had a talk with Łukasz we aren't able to check if system is sending version A of reset password email (in case AdminBundle is loaded) or version B (when there's only ApiBundle). He advised me to write integration tests directly inside `AdminBundle` and `ApiBundle`. If you check the last assert you'll notice we're checking against different strings in an email's content.